src/components/message.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { Link } from "gatsby"
```

Comment:
Codacy found an issue: ['Link' is defined but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5837031)